<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>decorDialog</class>
 <widget class="QDialog" name="decorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Внешний вид</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QCheckBox" name="chkFullScreenMainWindow">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="chkMaximizeMainWindow">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="sbFontSize">
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblFontSize">
     <property name="text">
      <string>Шрифт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblStyle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Стиль</string>
     </property>
     <property name="buddy">
      <cstring>cmbStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblStandartPalette">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Палитра из стиля</string>
     </property>
     <property name="buddy">
      <cstring>chkStandartPalette</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFontComboBox" name="cmbFontName"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblFullScreenMainWindow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Полно&amp;экранный режим</string>
     </property>
     <property name="buddy">
      <cstring>chkFullScreenMainWindow</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblMaximizeMainWindow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Максимизировать</string>
     </property>
     <property name="buddy">
      <cstring>chkMaximizeMainWindow</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cmbStyle"/>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkStandartPalette">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="chkCRBWidthUnlimited">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lblCRBWidthUnlimited">
     <property name="text">
      <string>Не ограничивать ширину выпадающих списков</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbStyle</tabstop>
  <tabstop>chkStandartPalette</tabstop>
  <tabstop>chkMaximizeMainWindow</tabstop>
  <tabstop>chkFullScreenMainWindow</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>decorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>decorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
